specs images are generated during the build and not disted

--- origsrc/libX11-1.3.4/specs/troffrules.in	2010-06-03 14:59:00.000000000 -0500
+++ src/libX11-1.3.4/specs/troffrules.in	2010-08-02 22:19:57.874315600 -0500
@@ -45,7 +45,7 @@ MOSTLYCLEANFILES = index.*
 # Install html generated images for specs
 install-data-local:
 	test -z "$(imagesdir)" || $(mkdir_p) "$(DESTDIR)$(imagesdir)"
-	@d="$(srcdir)/images/"; \
+	@d="images/"; \
 	list=`ls $$d`; \
 	for p in $$list; do \
 	  echo " $(INSTALL_DATA) '$$d$$p' '$(DESTDIR)$(imagesdir)/$$p'"; \
